packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -516,6 +516,7 @@ export default class Meeting extends StatelessWebexPlugin {
   resourceUrl: string;
   selfId: string;
   state: any;
+  webexMeetingId: string;
```

Comment:
this doesn't seem right. webexMeetingId is not a field on the Meeting object, at least I don't see it being used like that anywhere in this PR